resources/js/bots/wallet/bot.js

```diff
@@ -435,6 +450,36 @@ function previewSend(params, context) {
         markup = [firstRow];
     }
 
+    if (!(context["handler-data"]
+          && context["handler-data"]["status"] === "confirmed")) {
+        var pendingRow = status.components.text(
+            {
+                style: {
+                    color: "#9199a0",
+                    fontSize: 12,
+                    lineHeight: 18
+                }
+            },
+            I18n.t('send_transaction_pending')
+        ); 
+        markup.push(pendingRow);
+    }
+
+    if (context["handler-data"] 
+        && context["handler-data"]["status"] === "failed") {
+        var errorRow = status.components.text(
+            {
+                style: {
+                    color: "red",
+                    fontSize: 12,
+                    lineHeight: 18
+                }
+            },
+            I18n.t('send_transaction_errors') + ": " + context["handler-data"]["error"]
```

Comment:
<img width="226" alt="screen shot 2017-09-15 at 08 22 37" src="https://user-images.githubusercontent.com/2364994/30467686-27ac6360-99ef-11e7-9054-7c3ba56eb727.png">
We need to stringify error or exctract error message from `context["handler-data"]["error"]`

Reply:
error.message is a string:
> 'Invalid JSON RPC response: {"error":{"code":-32603,"transaction sending timed out":"transaction sending timed out"},"id":46,"jsonrpc":"2.0"}' 

which is confusing for end users as well... maybe better just leave 'transaction failed' instead of 'transaction pending' when error happened